Make systemd-networkd default network manager and DHCPv4/DHCPv6 client (#1925)
scaleway/docs-content: tutorials/dhcpcd-dedibox/index.mdx

[dhcpcd](https://roy.marples.name/projects/dhcpcd/) is an open-source DHCP client, supporting both IPv4, and IPv6. The feature-rich software can be used to configure and automate network-related parameters of your computer using the [DHCP protocol](http://en.wikipedia.org/wiki/Dynamic_Host_Configuration_Protocol).

<Message type="note">
Roy Marples' dhcpcd (DHCP client daemon) is not the same as the discontinued Internet Systems Consortium's dhcpd (DHCP (server) daemon).
</Message>

In this tutorial, you will learn how to configure your IPv6 subnet with DHCPCD on a Scaleway Dedibox dedicated server.

<Macro id="iam-requirements" />

<Message type="requirement">
  - You have an account and are logged into the [Dedibox console](https://console.online.net)
  - You have a [Dedibox&reg; dedicated server](https://www.scaleway.com/en/dedibox)
  - You have root access to the server
</Message>

## Installing DHCPCD

1. Change into the root account of your server by using `su` and the password of the `root` user.

> 2. 

Reviewer: remove extra number here and in the lines that follow
```suggestion
```

Author: Ok

> 2. 
>   1. For Debian & Ubuntu, update the `apt` package manager repositories, then upgrade the software already installed on the machine to the latest version available in the repositories: 

Reviewer 2: ```suggestion
2. For Debian & Ubuntu, update the `apt` package manager repositories, then upgrade the software already installed on the machine to the latest version available in the repositories: 
```

Author: Ok done

    ```
    apt update && apt upgrade -y
    ```

>   2. For Arch Linux, update the `pacman` package manager repositories.
>     ```
>     pacman -Syu
>     ```
> 3.
>   1. For Debian & Ubuntu, install DHCPCD using the `apt` package manager:

Reviewer 2: ```suggestion
3. For Arch Linux, update the `pacman` package manager repositories.
    ```
    pacman -Syu
    ```
4. For Debian & Ubuntu, install DHCPCD using the `apt` package manager:
```

Author: Ok done

    ```
    apt install dhcpcd5
    ```

> 3.
>   2. For Arch Linux, install DHCPCD using the Arch Linux package manager:

Reviewer 2: ```suggestion
5. For Arch Linux, install DHCPCD using the Arch Linux package manager:
```
